Resources/views/reset_password/check_email.html.twig

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+        Indien je e-mailadres bij ons bekend is, ontvang je van ons een e-mail met een link om je wachtwoord te
+        resetten.
+        <br>
+        <br>
+        Geen e-mail ontvangen? Controleer je spam folder of probeer het opnieuw.
+    </p>
+
+    <a href="./">Naar het inlogscherm</a>
```

Comment:
waarom ./ ???

Reply:
webstack_user_security_login gaf die een foutmelding aangezien wij app_security_login gebruiken, had dit al verbeterend in de portal maar hier nog niet